.github/templates/main.yml

```diff
@@ -44,21 +44,10 @@ steps:
 #@ end
 
 #@ def setupWin81SDK():
-  - name: Check Win8.1 SDK Cache
```

Comment:
This is unrelated to the fixes - I just noticed that downloading the SDK takes a few seconds, so caching is not really helpful